crates/forge/src/running.rs

```diff
@@ -68,17 +70,28 @@ fn build_hints_dict<'b>(
     (hints_dict, string_to_hint)
 }
 
-#[allow(clippy::too_many_arguments)]
+pub(crate) async fn blocking_run_from_test(
```

Comment:
Misleading name, it is not actually blocking if it is `async`. Moreover I think we can just copy paste the line from this func to the two places it is used, but that's up to you

Reply:
I prefer to leave it as a separate function, because I use it in tokio::select!, and it already has quite a complex structure. Any idea for better name?

Reply:
Actually it is blocking since it uses `spawn_blocking` 🤦  mb